nucypher/cli/main.py

```diff
@@ -70,20 +71,23 @@ def nucypher_cli():
 
 ENTRY_POINTS = (
 
-    # Characters
-    alice.alice,        # Author of Policies
-    bob.bob,            # Builder of Capsules
-    enrico.enrico,      # Encryptor of Data
-    ursula.ursula,      # Untrusted Re-Encryption Proxy
+    # Characters & Actors
+    alice.alice,    # Author of Policies
+    bob.bob,        # Builder of Capsules
+    enrico.enrico,  # Encryptor of Data
+    ursula.ursula,  # Untrusted Re-Encryption Proxy
+    stake.stake,    # Stake Management
 
     # Utility Commands
-    dao.dao,            # NuCypher DAO
-    stake.stake,        # Stake Management
     status.status,      # Network Status
-    felix.felix,        # Faucet
+    dao.dao,            # NuCypher DAO
     multisig.multisig,  # MultiSig operations
-    worklock.worklock,         # WorkLock
-    cloudworkers.cloudworkers  # Remote Worker node management
+    worklock.worklock,  # WorkLock
+    felix.felix,        # Faucet
+
+    # Application commands
+    cloudworkers.cloudworkers,  # Remote Worker node management
+    contacts.contacts,          # Character "card" management
```

Comment:
Is this more of a utility command?

Reply:
Hmm - yes I think it is. :heavy_check_mark: